util/export_gkube_images.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# export discovery image
+discovery_pod="$(gkube kubectl get pods | grep discovery  | head -n 1 | awk '{print $1}')"
+discovery_image=$(gkube kubectl describe pod "$discovery_pod"  | grep docker.greenkeytech.com/discovery | awk '{print $NF}' | head -n 1)
```

Comment:
```suggestion
discovery_image=$(kubectl get deploy gk-discovery -o go-template='{{range .spec.template.spec.containers}}{{if eq "discovery" .name}}{{.image}}{{end}}{{end}}')
```

Reply:
This needs the gkube command since clients don't install kubectl globally

Reply:
sorry, `gkube kubectl ...`